cfgov/unprocessed/apps/searchgov/css/main.scss

```diff
@@ -0,0 +1,20 @@
+@use 'sass:math';
+@use '@cfpb/cfpb-design-system/src/abstracts' as *;
+
+.search-result {
+    > h2 {
+        font-weight:400
```

Comment:
Run `yarn lint` … which I think will tidy this up by adding the space and semicolon here, and adding a newline at the end of the file.

Reply:
Thanks! Fixed in 65e45b8.